addons/account_peppol/models/account_journal.py

```diff
@@ -27,7 +27,7 @@ def action_peppol_ready_moves(self):
         return {
             'name': _("Peppol Ready invoices"),
             'type': 'ir.actions.act_window',
-            'view_mode': 'list',
+            'view_mode': 'list,form',
```

Comment:
I noticed that you can't open invoices from Peppol Ready view, so this is a fix for that

Reply:
So it should be in a different commit then :)